### DISCUSSION TITLE
Importing LCIA methods from the standard Excel template

### DISCUSSION THREAD

**Discussion by nizaanth** [brightway-lca/brightway-documentation#110] [Documentation Request]
Hi,

I am trying to work on a small project that involves Importing LCIA methods from excel file.  However, this section in the brightway documentation page seems to be badly formatted. Any help/fix on this issue/documentation would be grateful. Thanks.

**Comment by michaelweinold** [comment]
Hello, hello! Thank you for getting in touch. I'll check over the weekend. If you don't hear from me by next Wednesday (Tuesday is a bank holiday in 🇨🇭), ping me again in this thread.

**Comment by michaelweinold** [comment]
> However, this section in the brightway documentation page seems to be badly formatted.

Please provide the link to the section.

**Reply by nizaanth** [reply]
https://docs.brightway.dev/en/latest/content/introduction/introduction.html#importing-lcia-methods-from-the-standard-excel-template

**Reply by michaelweinold** [reply]
Thank you - I've added an issue over the weekend:

 - https://github.com/brightway-lca/brightway-documentation/issues/114

I expect this will be fixed in one week.
